static/js/unread.js

```diff
@@ -623,6 +682,11 @@ export function stream_has_any_unread_mentions(stream_id) {
     return streams_with_mentions.has(stream_id);
 }
 
+export function topic_has_any_unread_mention(stream_id, topic) {
```

Comment:
Renamed this to `topic_has_any_unread_mentions` to better parallel the existing `stream_has_any_unread_mentions`